htdocs/expedition/class/expedition.class.php

```diff
@@ -1806,6 +1806,14 @@ function setClosed()
 		{
 			// TODO: Add option/checkbox to set order billed if 100% of order is shipped
 			$this->statut=2;
+			
+			// Call trigger
+			$result=$this->call_trigger('SHIPPING_CLOSED',$user);
```

Comment:
If you introduce a trigger here, it means you can have other sql orders, so you must introduce a begin and commit/rollback into the function setClose to have all sql request in same transation. Same for other methods.